views/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Pokédex</title>
+    <link rel = "shortcut icon" href = "https://raw.githubusercontent.com/lawandothman/pokedex/features/favicon.ico">
     <link rel = "stylesheet" type = "text/css" href = "/style.css">
```

Comment:
You usually don't have spaces between the key and space and the value so:
```html
<link rel="shortcut">
<!-- rather than -->
<link rel = "shortcut">
```